src/cowmachine_response.erl

```diff
@@ -40,19 +40,23 @@ server_header() ->
     case application:get_env(cowmachine, server_header) of
         {ok, Server} -> z_convert:to_binary(Server);
         undefined ->
-            {ok, Version} = application:get_key(cowmachine, vsn),
-            <<"CowMachine/", (z_convert:to_binary(Version))/binary>>
+            case application:get_key(cowmachine, vsn) of
+                {ok, Version} -> <<"CowMachine/", (z_convert:to_binary(Version))/binary>>;
+                undefined -> <<"CowMachine">>
```

Comment:
Might this happen because the cowmachine application was not started in the test?

Reply:
Ah, yes, it needs to list the other apps: cowboy, lager, zotonic_stdlib.

Was somehow never added.

Reply:
Yes, you are right... When it is started you get a version number.

`CowMachine/1.1.1+build.66.refb14a3be`

We may want to keep the handling of `undefined` here, because it is a valid return value from `application:get_key/2`